source-install.yml

```diff
@@ -1,9 +1,13 @@
 ---
 - hosts: all
   vars:
+    package_retries: "5"
     pulp_source_dir: "/home/vagrant/devel/pulp"
     pulp_plugin_source_dir: "/home/vagrant/devel/pulpcore-plugin"
     pulp_secret_key: "unsafe_default"
+    developer_user: "vagrant"
```

Comment:
Source install will definitely be used by non-vagrant users. IMO, anything vagrant specific should should be in the Vagrantfile.source.example (maybe change that to Vagnrantfile.dev.example)